src/mdw/java/com/lambdaherding/edi/mdw/ch02/Calculate.java

```diff
@@ -1,4 +1,4 @@
-package com.lambdaherding.edi.ch02;
+package com.lambdaherding.edi.mdw.ch02;
```

Comment:
I was going to update it but decided to skip ch 2. I forgot all that was on this branch.